src/hooks/useFormSubmissionAutoSaveState.ts

```diff
@@ -64,27 +64,36 @@ export default function useFormSubmissionAutoSaveState({
         model: SubmissionTypes.S3SubmissionData['submission'],
         lastElementUpdated?: FormElement,
       ) => {
-        console.log('Auto saving...')
-        autoSaveService
-          .upsertAutoSaveData(definition.id, autoSaveKey, model)
-          .then(() => {
-            if (lastElementUpdated) {
-              return autoSaveService.upsertAutoSaveData<FormElement>(
-                definition.id,
-                `LAST_ELEMENT_UPDATED_${autoSaveKey}`,
-                lastElementUpdated,
-              )
+        if (!formIsDisabled) {
+          switch (lastElementUpdated?.type) {
+            case 'summary':
+            case 'calculation':
+            case 'captcha': {
```

Comment:
This wont prevent the `resetRecaptchas()` function from running as it does not set the `lastElementUpdated` property when updating the submission.

Reply:
yeah but this function will run nearly 10 seconds later, by which time it will have lost the race.

Reply:
hmmmm but memoing....

Reply:
Just tested with a 5 second delay on disabling the form and it worked.
With a 15 second delay it obviously triggered autosave. So I'm happy with this.

Reply:
As discussed, im happy if you are happy.